bugs/PENDING_REVIEW/0433-BAD-IMAGE-Pixel5-29b7a44e-no-opt-test/reduced_1_result/test.amber

```diff
@@ -0,0 +1,719 @@
+               OpBranch %72
+         %72 = OpLabel
+         %76 = OpAccessChain %17 %10 %36
+         %77 = OpLoad %6 %76
+         %78 = OpFOrdLessThan %23 %77 %27
+               OpSelectionMerge %80 None
+               OpBranchConditional %78 %79 %80
+         %79 = OpLabel
+         %82 = OpLoad %6 %54
+         %83 = OpFSub %6 %81 %82
+               OpStore %54 %83
+               OpBranch %80
+         %80 = OpLabel
+         %84 = OpAccessChain %17 %10 %33
+         %85 = OpLoad %6 %84
+         %86 = OpFOrdLessThan %23 %85 %27
+               OpSelectionMerge %88 None
+               OpBranchConditional %86 %87 %88
+         %87 = OpLabel
+         %89 = OpLoad %6 %54
+         %90 = OpFNegate %6 %89
+               OpStore %54 %90
+               OpBranch %88
+         %88 = OpLabel
+         %91 = OpLoad %6 %54
+               OpReturnValue %91
+               OpFunctionEnd
+         %15 = OpFunction %7 None %13
+         %14 = OpFunctionParameter %8
+         %16 = OpLabel
+         %94 = OpVariable %17 Function
+         %96 = OpVariable %8 Function
+        %100 = OpVariable %17 Function
+        %103 = OpVariable %17 Function
+        %104 = OpVariable %8 Function
+               OpStore %94 %95
+         %97 = OpLoad %7 %14
+         %99 = OpFSub %7 %97 %98
+               OpStore %96 %99
+        %101 = OpLoad %7 %96
+        %102 = OpExtInst %6 %1 Length %101
+               OpStore %100 %102
+        %105 = OpLoad %7 %96
+               OpStore %104 %105
+        %106 = OpFunctionCall %6 %11 %104
+               OpStore %103 %106
+        %107 = OpLoad %6 %100
+        %108 = OpLoad %6 %103
+        %109 = OpCompositeConstruct %7 %107 %108
+               OpReturnValue %109
+               OpFunctionEnd
+END
+
+# uniforms for reference
+
+# tex
+SAMPLER reference_tex
+
+
+BUFFER reference_framebuffer FORMAT B8G8R8A8_UNORM
+
+PIPELINE graphics reference_pipeline
+  ATTACH reference_vertex_shader
+  ATTACH reference_fragment_shader
+  FRAMEBUFFER_SIZE 256 256
+  BIND BUFFER reference_framebuffer AS color LOCATION 0
+  BIND BUFFER default_texture AS combined_image_sampler SAMPLER reference_tex DESCRIPTOR_SET 0 BINDING 0
+END
+CLEAR_COLOR reference_pipeline 0 0 0 255
+
+CLEAR reference_pipeline
+RUN reference_pipeline DRAW_RECT POS 0 0 SIZE 256 256
+
+
+SHADER vertex variant_vertex_shader PASSTHROUGH
+
+# variant_fragment_shader is derived from the following GLSL:
+# #version 320 es
+#
+# precision highp float;
+#
+# layout(location = 0) out vec4 _GLF_color;
+#
+# layout(set = 0, binding = 0) uniform sampler2D tex;
+#
+# float ReallyApproxNormalizedAtan2(vec2 v)
+# {
+#  float pi2 = 1.0 / (355.0 / 113.0);
+#  if(length(v) < 0.001)
+#   {
+#    return 0.0;
+#   }
+#  vec2 a = abs(v);
+#  float z;
+#  if(a.y > a.x)
+#   {
+#    z = a.x / a.y;
+#   }
+#  else
+#   {
+#    if((gl_FragCoord.y < length(vec3(0.0, 0.0, 0.0))))
+#     {
+#      do
+#       {
+#        discard;
+#       }
+#      while(true);
+#     }
```

Comment:
Can you add a comment to explain that the guard will always evaluate to false, and that this block of code is the sole difference between the reference and the variant?

Reply:
I haven't worked on these things in a while, so correct me if I'm wrong, but are you sure it's this file that you want changed? The variant shader.frag in `/0433-BAD-IMAGE-Pixel5-29b7a44e-no-opt-test/reduced_1/variant/` contains the comments. This shader is then used by generate_cts_test.py to generate the .amber file which in this case is called `stable-sampler-polar-warp-do-discard-false-if.amber`. This is the .amber file that contains the comments as well and will eventually be submitted to CTS, correct? 

I am not sure if `/reduced_result_1/` is actually used for anything in this case. Unless there has been a misunderstanding and the .amber file that will be submitted to CTS is supposed to be generated from the files that reside in `/reduced_result_1/` and NOT from the ones in `/reduced_1/`?

Reply:
You're correct: this isn't the file that should be changed, but from a reviewing point of view it's easiest if I comment on this file (as it is the file that will eventually land in CTS), and then you work out where to make the chnages that will lead to this file changing. Do you see what I mean?

Reply:
I think I'm missing something here... I get what you mean by making the changes somewhere else, the changes will be in:

`/reduced_1/reference/shader.frag` <- To remove the redundant line.
And
`/reduced_1/reference/shader.frag` <- To add the comments. BUT this file already contains the comments.

And so does `stable-sampler-polar-warp-do-discard-false-if.amber`, as this is the file that has been generated by the python script, using the files in `/reduced_1/` indicated on line 42: `source_dir=bug_dir / "reduced_1"` in `generate_cts_test.py`. It would make sense that this is the file that will be submitted to CTS, as it's named according to the spec rather than it being called test.amber.
What makes me feel like I'm missing something here is that you have commented on `reduced_result_1/test.amber` and wrote:
> (as it is the file that will eventually land in CTS)

Pointing to the file (`/reduced_result_1/test.amber`) that will not change regardless of changing the lines or commenting in `/reduced_1/*/shader.frag`

Reply:
To clarify, this is the wrong amber file. This is a temporary amber file created as part of running the automatically reduced shader, with no manual changes, with no copyright header, no comments, etc.

Reply:
Ah, yes! Sorry; I had intended to comment on this file:

bugs/PENDING_REVIEW/0433-BAD-IMAGE-Pixel5-29b7a44e-no-opt-test/stable-sampler-polar-warp-do-discard-false-if.amber

Sorry for the confusion.

I believe the comments I've made above apply equally well in the context of that file, so are you OK to go ahead and make them?

Reply:
No problem. And yes, I'll make the changes.